homeassistant/components/tradfri/cover.py

```diff
@@ -0,0 +1,139 @@
+"""Support for IKEA Tradfri covers."""
+import logging
+
+from homeassistant.components.cover import CoverDevice, ATTR_POSITION
+from homeassistant.core import callback
+from . import DOMAIN as TRADFRI_DOMAIN, KEY_API, KEY_GATEWAY
+from .const import CONF_GATEWAY_ID
+
+_LOGGER = logging.getLogger(__name__)
+
+IKEA = "IKEA of Sweden"
```

Comment:
We use the `IKEA` constant in many platforms. We can move it to const.py in the package.

Reply:
Hmm, looking more closely at it, I'm not sure it's being used at all. I'll remove it from all files.